core/adapters/snowflake.ts

```diff
@@ -80,4 +81,24 @@ export class SnowflakeAdapter extends Adapter implements IAdapter {
       table.target
     )} as ${table.query}`;
   }
+
+  public insertInto(
```

Comment:
afaict this is identical to bigquery. Can we re-use a function? I realize this code is all a mess, but a good opportunity to clean it up.

Reply:
Ah, I did that originally but they do have slightly different syntax, e.g. `merge into` rather than just `merge`. With BQ now having the partition filter I think it makes sense not to have a generic function